be/src/storage/tablet_updates.cpp

```diff
@@ -1806,31 +1805,30 @@ Status TabletUpdates::perform_linked_schema_change(int64_t request_version, Tabl
     index.unload();
     update_manager->index_cache().release(index_entry);
     _tablet.set_tablet_state(TabletState::TABLET_RUNNING);
-    LOG(INFO) << "perform_linked_schema_change finish tablet:" << _tablet.tablet_id()
-              << " version:" << this->max_version() << " #pending:" << _pending_commits.size();
+    LOG(INFO) << "schema_change_linked finish tablet:" << _tablet.tablet_id() << " version:" << this->max_version()
+              << " #pending:" << _pending_commits.size();
     return Status::OK();
 }
 
-Status TabletUpdates::perform_directly_schema_change(int64_t request_version,
-                                                     const std::shared_ptr<Tablet>& base_tablet,
-                                                     vectorized::ChunkChanger* chunk_changer) {
+Status TabletUpdates::schema_change_directly(int64_t request_version, const std::shared_ptr<Tablet>& base_tablet,
+                                             vectorized::ChunkChanger* chunk_changer) {
     DCHECK(_tablet.tablet_state() == TABLET_NOTREADY)
-            << "perform_directly_schema_change is only allowed in schema change process";
-    LOG(INFO) << "perform_directly_schema_change start tablet:" << _tablet.tablet_id()
-              << " request_version:" << request_version << " #pending:" << _pending_commits.size();
+            << "schema_change_directly is only allowed in schema change process";
+    LOG(INFO) << "schema_change_directly start tablet:" << _tablet.tablet_id() << " request_version:" << request_version
```

Comment:
If schema_change_directly is a verb, this place should be replaced as a noun.

Reply:
if "xxx start",  the "xxx" can be a noun,
if "start xxx", the "xxx" can be a verb.
a proposal:
```
LOG(INFO) << "start link_from. "
              << " tablet_id:" << _tablet.tablet_id() << " request_version:" << request_version
              << " #pending:" << _pending_commits.size();
```